integration-tests/gauntlet_starknet.go

```diff
@@ -0,0 +1,161 @@
+package integration_tests
```

Comment:
This file redeclares a bunch of code and types that should be already available elsewhere, and are common across integrations. Running it is our first priority, but we should also figure out how to improve this situation.

@smickovskid, please keep this in mind and try to reuse it if there is existing code available. If not, let's try to split the code into:

- common code (shared across integrations)
- custom StarkNet-specific code

Let's then put all common code in `relayer/pkg/common` and in-line document there our ad-hoc suggestion where should this code be moved. Then we can go back and do cleanup until we remove the `common` package altogether.

cc @archseer @aalu1418 @sdrug

Reply:
Adjusted and re-structured